.gitmodules

```diff
@@ -1,54 +1,57 @@
-[submodule "folly"]
-	path = folly/src
-	url = https://github.com/facebook/folly.git
-[submodule "thrift/src"]
-	path = thrift/src
-	url = https://github.com/facebook/fbthrift.git
-[submodule "proxygen/src"]
-	path = proxygen/src
-	url = https://github.com/facebook/proxygen
-[submodule "mcrouter/src"]
-	path = mcrouter/src
-	url = https://github.com/facebook/mcrouter
-[submodule "re2/src"]
-	path = re2/src
-	url = https://github.com/google/re2.git
-[submodule "squangle/squangle"]
-	path = squangle/src
-	url = https://github.com/facebook/squangle.git
-[submodule "wangle/src"]
-	path = wangle/src
-	url = https://github.com/facebook/wangle
-[submodule "webscalesqlclient/mysql-5.6"]
-	path = webscalesqlclient/mysql-5.6
-	url = https://github.com/facebook/mysql-5.6.git
-[submodule "brotli/src"]
-	path = brotli/src
-	url = https://github.com/google/brotli
-[submodule "double-conversion"]
-	path = double-conversion
-	url = https://github.com/google/double-conversion
-[submodule "fastlz/src"]
-	path = fastlz/src
-	url = https://github.com/ariya/FastLZ
-[submodule "libafdt/src"]
-	path = libafdt/src
-	url = https://github.com/facebook/libafdt
+[submodule "folly"]
+    path = folly/src
+    url = https://github.com/facebook/folly.git
+[submodule "thrift/src"]
+    path = thrift/src
+    url = https://github.com/facebook/fbthrift.git
+[submodule "proxygen/src"]
+    path = proxygen/src
+    url = https://github.com/facebook/proxygen
+[submodule "mcrouter/src"]
+    path = mcrouter/src
+    url = https://github.com/facebook/mcrouter
+[submodule "re2/src"]
+    path = re2/src
+    url = https://github.com/google/re2.git
+[submodule "squangle/squangle"]
+    path = squangle/src
+    url = https://github.com/facebook/squangle.git
+[submodule "wangle/src"]
+    path = wangle/src
+    url = https://github.com/facebook/wangle
+[submodule "webscalesqlclient/mysql-5.6"]
+    path = webscalesqlclient/mysql-5.6
+    url = https://github.com/facebook/mysql-5.6.git
+[submodule "brotli/src"]
+    path = brotli/src
+    url = https://github.com/google/brotli
+[submodule "double-conversion"]
+    path = double-conversion
+    url = https://github.com/google/double-conversion
+    ignore = dirty
```

Comment:
That would require submitting pull requests to google/double-conversion, facebook/libafdt and nih-at/libzip.  There's no reason to not just ignore dirty changes in those submodules since we never expect to change them locally.